frontend/src/shared/components/tasque-task-creation/task-creation.component.html

```diff
@@ -1,23 +1,22 @@
 <tasque-sidebar name="task-creation" [btnText]="btnText" [btnClass]="btnClass">
   <div class="task-creation-conteiner">
     <div class="task-creation-header">
-      <h4 class="task-creation-header-title" style="color: black">Create issue</h4>
-      <div class="task-creation-header-button">
-        <tasque-button class="btn mini disabled" text="Import issues"></tasque-button>
-        <tasque-button class="btn icon" [icon]="faExpeditedssl"></tasque-button>
-      </div>
+      <h4 class="task-creation-header-title" style="color: black">
+        Create issue
+      </h4>
     </div>
     <div class="task-creation-body">
       <div class="task-creation-body-wrapper">
         <div class="body-title">
           <span>Basic info</span>
         </div>
-        <hr>
+        <hr />
```

Comment:
This is a selector that doesn't has a closure item, like basic input. This will be rendered as fine as before but for the future